makefiles/stdio.inc.mk

```diff
@@ -53,5 +65,8 @@ endif
 
 ifneq (,$(filter stdio_semihosting,$(USEMODULE)))
   USEMODULE += xtimer
+  ifneq (,$(filter picolibc,$(USEMODULE)))
+    USEMODULE += picolibc_stdout_buffered
+  endif
```

Comment:
Does it make sense to replace all instances of this check with a single check at the bottom of the file?

```
ifneq (,$(filter picolibc,$(USEMODULE)))
  ifneq (,$(filter stdio_cdc_acm stdio_ethos slipdev_stdio stdio_semihosting,$(USEMODULE)))
    USEMODULE += picolibc_stdout_buffered
  endif
endif
```

Reply:
Good idea!